packages/semantic-ui/test/Form.test.tsx

```diff
@@ -188,9 +188,10 @@ describe("single fields", () => {
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
-  test("checkbox field", () => {
+  test("checkbox field with label", () => {
     const schema: RJSFSchema = {
       type: "boolean",
+      title: "test",
```

Comment:
I know this might be asking too much, but this test is the same in all the other themes except `core`. Would you be willing to update the other tests (antd, bootstrap4, chakra-ui, fluent-ui, material-ui, and mui) to add the `title`?